doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -452,3 +452,21 @@ of ``/sys/block/<drive>/queue/optimal_io_size``.
 .. confval:: bluestore_min_alloc_size_ssd
 .. confval:: bluestore_use_optimal_io_size_for_min_alloc_size
 
+DSA (Data Streaming Accelerator Usage)
+======================================
+
+If you want to use the DML library to drive DSA device for offloading
+read/write operations on Persist memory in Bluestore. You need to install
+`DML`_ and `idxd-config`_ library in your machine with SPR (Sapphire Rapids) CPU.
+Refer following site for more details.
+
+.. _DML: https://github.com/intel/DML
+.. _idxd-config: https://github.com/intel/idxd-config`
```

Comment:
remove the trailing "`". the link in https://ceph--44230.org.readthedocs.build/en/44230/rados/configuration/bluestore-config-ref/ brings me to https://github.com/intel/idxd-config%60 which is a 404.

Reply:
ack